code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -115,6 +115,7 @@
 	..()
 
 /obj/item/clothing/suit/space/hardsuit/attackby(obj/item/I, mob/user, params)
+	. = ..()
```

Comment:
Only do this if the two if conditions below don't proc

Reply:
So a regular return ..() in the end, then? Lemme replace it (though I've checked and both adding and removing the jetpack worked).